package.json

```diff
@@ -10,6 +10,14 @@
     "test": "jest"
   },
   "dependencies": {
+
```

Comment:
"integrity": "sha512-CL/29uS74AwreI/f2oz2hLTW7ZqVeV5+gxFeGudzQrgkCytrHw33G4KbnQOrRlAEzzAFXi7dDLMC9zhWcVpzmw==",
      "license": "MIT",
      "peerDependencies": {
        "react": ">=16.8"
>>>>>>> ad72a101ef61f279d739d279efed28ae8db8d173
      }
    },
=======

>>>>>>> 59c434fe7f2fc84fa9e66980e9a85066d73d1bb9
    "node_modules/util-deprecate": {
      "version": "1.0.2",
      
Make sure the merging you are doing is proper and correct without leaving this commit hash marks. Shows merge was not completed properly.